CardData.py

```diff
@@ -182,12 +113,14 @@ def create_card_json(gwent_data_helper, patch):
         card['categoryIds'] = []
         categoriesSum = int(template.find('Categories').find('e0').attrib['V']);
         # XML Card category is the sum of all categories of the card
-        for category in sorted(CATEGORIES, reverse=True):
-            if categoriesSum - category >= 0:
-                categoriesSum -= category
-                card['categoryIds'].append(CATEGORIES[category])
-            if categoriesSum == 0:
-                break
+        for category in range(LAST_CATEGORY, 1, -1):
+          categoryPower = 2**category
+          if categoriesSum - categoryPower >= 0:
+            categoriesSum -= categoryPower
+            if category in CATEGORIES:
+              card['categoryIds'].append("card_category_" + str(category))
+          if categoriesSum == 0:
+            break
```

Comment:
Can you change this block to use 4 spaces for indentation please to match the rest of the project.